hc/api/tests/test_ping.py

```diff
@@ -63,11 +65,32 @@ def test_it_reads_forwarded_protocol(self):
                             HTTP_X_FORWARDED_PROTO="https")
         ping = Ping.objects.latest("id")
         ### Assert the expected response status code and ping's scheme
+        self.assertEqual(r.status_code, 200)
+        self.assertEqual(ping.scheme, "https")
 
     def test_it_never_caches(self):
         r = self.client.get("/ping/%s/" % self.check.code)
         assert "no-cache" in r.get("Cache-Control")
 
     ### Test that when a ping is made a check with a paused status changes status
+    def test_when_a_ping_is_made_check_status_changes(self):
+        ip = "1.1.1.1, 2.2.2.2"
+        r = self.client.get("/ping/%s/" % self.check.code,
```

Comment:
this ping is made to a check with the status new, so the instruction hasn't been followed

Reply:
@Jak-Sidious @pndemo Any updates on this?

Reply:
I had totally spaced out on this, please give me a few minutes to fix this